Doc/faq/design.rst

```diff
@@ -584,9 +584,9 @@ exhaustive test suites that exercise every line of code in a module.
 An appropriate testing discipline can help build large complex applications in
 Python as well as having interface specifications would.  In fact, it can be
 better because an interface specification cannot test certain properties of a
-program.  For example, the :meth:`list.append` method is expected to add new elements
+program.  For example, the :meth:`!list.append` method is expected to add new elements
```

Comment:
I wonder, is it worth linking to the table in https://docs.python.org/3/library/stdtypes.html#mutable-sequence-types instead?

That's as close to a definition we have, but I'm not sure if it's better than this `!`.

Reply:
The challenge is you'd loose the :func: styles, as you'd need to use :ref: -- do-able, but I suppose I assumed knowledge of what `append()` does?

A

Reply:
I'm not too bothered about the styles, but yeah, `append()` is one of the basics :) This is fine, thank you!